package.json

```diff
@@ -12,7 +12,12 @@
     "test": "jest",
     "test:watch": "jest --watchAll",
     "test:coverage": "npm run test -- --coverage",
-    "test:open": "cypress open"
+    "test:open": "cypress open",
+    "lint:fix": "eslint ./ --cache --ignore-pattern .gitignore --fix",
+    "lint": "eslint ./ --cache --ignore-pattern .gitignore",
+    "build": "babel src/ -d lib/",
```

Comment:
What is the purpose of this script?

Reply:
Sorry by mistake added but, forget to remove :)